src/site-server.ts

```diff
@@ -149,4 +159,34 @@ export class SiteServer {
 			.catch( Sentry.captureException )
 			.finally( () => window.destroy() );
 	}
+
+	async executeWpCliCommand( args: string ): Promise< WpCliResult > {
+		const projectPath = this.details.path;
+
+		if ( ! this.wpCliExecutor ) {
+			this.wpCliExecutor = new WpCliProcess( projectPath );
+			await this.wpCliExecutor.init();
+		}
+
+		const wpCliArgs = parse( args );
+
+		// The parsing of arguments can include shell operators like `>` or `||` that the app don't support.
+		const isValidCommand = wpCliArgs.every(
+			( arg: unknown ) => typeof arg === 'string' || arg instanceof String
+		);
+		if ( ! isValidCommand ) {
+			throw Error( `Can't execute wp-cli command with arguments: ${ args }` );
```

Comment:
Suggestion to increase clarity by avoiding contractions. Feel free to disregard. 

```suggestion
			throw Error( `Cannot execute wp-cli command with arguments: ${ args }` );
```